### PR TITLE
fix: Fixed spacing in alert modal

### PR DESCRIPTION
**Pull request by lyndsiWilliams**
### SUMMARY

This PR fixes the spacing under the "Message content" section of the alert modal.

Note: When I adjusted the spacing I noticed a few spots in the component that needed translating/cleanup so I added that to this PR as well.

#### BEFORE:
alertReportBefore

#### AFTER:
alertReportAfter

#### AFTER (extended)
alertReportAfterExtendedEdition

### TESTING INSTRUCTIONS

- Go to the Alerts & reports page and click the "+ ALERT" button
- Observe the spacing between elements

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by pkdotson — APPROVED** [review]

**Comment by eschutho** [comment]
@lyndsiWilliams This looks great!! Can we also left align it to either match up with the dashboard/chart select box or the email select box? Maybe @yousoph  or a designer my have more opinions there.

**Comment by rusackas** [comment]
@lyndsiWilliams @yousoph will there be any more actions happening on this PR, or should we merge it as-is?

**Comment by lyndsiWilliams** [comment]
> @lyndsiWilliams @yousoph will there be any more actions happening on this PR, or should we merge it as-is?

More actions for sure! @yousoph , @eschutho , Karan, and I all got together on zoom to check this out together and discuss the styling. We decided to, for now, just get everything aligned with the 16px border to clean it up a little more. I just pushed up the changes in [`this commit`](https://github.com/apache/superset/pull/22066/commits/af49885ce136d8c23fde80fc263d919bc83ed82b).

File: superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx [hunk 1]
```diff
@@ -1300,7 +1309,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
                     placeholder={t('Time in seconds')}
                     onChange={onTimeoutVerifyChange}
                   />
-                  <span className="input-label">seconds</span>
+                  <span className="input-label">{t('seconds')}</span>
```
[the remark this answers is not in the document]
Comment (lyndsiWilliams) [reply superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx]: You got it! Done in [`this commit`](https://github.com/apache/superset/pull/22066/commits/44a5883c12fd8d6d00d149723956a744dd87dc98). (I think I got them all!)

**Comment by lyndsiWilliams** [comment]
Closing and reopening to reset the ephemeral environment, things aren't showing up properly 🤔

**Comment by lyndsiWilliams** [comment]
The ephemeral environment doesn't seem to be working correctly for this PR, so visual testing will have to be done locally. This is how it should look:
alertModalPolish

**Comment by rusackas** [comment]
Ephemeral environment seems to be running OK, but I spotted a couple of styling issues there. I can't seem to upload an image here for some reason at the moment, so I'll Slack ya.

**Comment by rusackas** [comment]
Do we still have build/ephemeral concerns here, or is this ready for the final once-over?

**Review by AAfghahi — APPROVED** [review]
Looks good!

**Comment by lyndsiWilliams** [comment]
> Do we still have build/ephemeral concerns here, or is this ready for the final once-over?

I'm not sure what's going on with the ephemeral, I'd be happy to zoom and show this locally if that would increase confidence in this. I feel (and hope) that this is just a case of the ephemeral not working correctly.

File: superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx [hunk 2]
```diff
@@ -1044,14 +1051,50 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     setIsHidden(false);
   }
 
+  const SAVE_TEXT = t('Save');
```
Comment (eschutho) [inline superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx:1054]: @lyndsiWilliams are we going to move these into a `TRANSLATIONS` object?

Comment (lyndsiWilliams) [reply superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx]: Definitely! Done in [`this commit`](https://github.com/apache/superset/pull/22066/commits/c74039460331569ae9d29c6398fd2dd937a3971b).

**Review by eschutho — APPROVED** [review]
Looks great!

**Pull request merged**